extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -390,6 +384,16 @@ class AmpYoutube extends AMP.BaseElement {
 
     if (eventType == 'initialDelivery') {
       this.info_ = info;
+
+      if (info['duration'] !== undefined) {
+        // Video not available.
+        if (info['duration'] <= 0 && info['videoData'] &&
+            info['videoData']['title'] == '') {
+          this.removeIframe_();
+          this.toggleFallback(true);
+        }
```

Comment:
Occasionally (1 out of about 25 refreshes) this check is not triggered, leaving fallback never toggled when loading an invalid video. Seems not related to listener attachment. More likely that some other checks involving `initialDelivery` return first, so the code block is never reached.

Reply:
interesting. Does the `info` not arrive on those cases or `toggleFallback` does not do its thing?

If former, moving or duplication the check in `infoDelivery` message may help

Reply:
The message does not arrive, which is strange. Closing for now as more investigation is needed.